src/modules/pokemons/PokemonNameType.ts

```diff
@@ -0,0 +1,1155 @@
+export type PokemonNameType
```

Comment:
Should we also leave some commented code here for people to use to update the list?

something like:
```suggestion
/*
copy(`export type PokemonNameType
    = ${pokemonList.map(p => `'p.name'`).join('\n    | ')};`);
*/
export type PokemonNameType
```

Although I'm not sure if there is a way to force run ts and ignore any type errors 🤔

Reply:
That is the exact code I used :sweat_smile: (it leaves a bunch of unescaped `'` in names like Farfetch'd though)

Having that would be useful, we can get around the type errors when updating the list by making a temporary copy of PokemonList without any type restrictions, and run the code on that list instead.

Reply:
Ahh, forgot about those ones, we should just be able to use `${p.name.replace(/'/g, "\\'")}` to escape those.

Good point, that should do the trick.